kmm/functional_base.py

```diff
@@ -0,0 +1,38 @@
+from pydantic import BaseModel, Extra
+
+
+class FunctionalBase(BaseModel):
```

Comment:
Why do we need this? Remove unused methods?

Reply:
I like the `replace` method and the default config, but yeah I guess the rest isn't used

Reply:
At least remove the experimental methods imo